REQUIRE

```diff
@@ -0,0 +1,2 @@
+julia 0.3
+Compat
```

Comment:
more conservative bounds are better than too-loose bounds, but you should only set bounds for errors, not deprecation warnings